components/email/src/init.spec.js

```diff
@@ -733,3 +733,188 @@ describe("Email: Toggle email of sender/recipient", () => {
     });
   });
 });
+
+describe("Should Render Reply Button And Dispatch Event When Clicked", () => {
+  it("Should Render Reply Button When Passed A Message", () => {
+    cy.visit("/components/email/src/examples/reply-message.html");
```

Comment:
First of all, love that we have a bunch of these demo pages to showcase the features of the component.

However from a maintainability standpoint, I'd want the _tests_ to lean on the most minimal version of the component (eg. `cypress.html`) and do the component manipulation in this test file via `cy.intercept` for the manifest or `prop` changes. The consequence of testing these individual page files is that:

1. Every time we need to test a new use case/feature, we need to create an entirely new `.html` file and do the manipulation there - _we will probably lose track of all these html files_
2. When I need to update a breaking test, it'd be difficult to tell what's happening by looking at the test file alone - for velocity sake, the less files we need to have open the quicker it is to debug (given that everything is appropriately encapsulated).
3. Some confusion about where/when to expand on code - If I'm adding onto a feature to `reply-message` for example, do I change `reply-message.html`, do I create a new `html` file? I'd think it'd be much easier to add a new test and manipulate everything there.

Totally understand if this sounds nitpicky but a major reason for our test flakiness was because every dev had a different way to test the components making it increasingly unmaintainable.

Reply:
I agree with you @krisgardiner. I think the component manipulation should be in the test itself--I left a similar comment